x-pack/plugins/ml/public/cases/index.tsx

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import type { CasesUiSetup } from '@kbn/cases-plugin/public/types';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { EuiButtonIcon, EuiDescriptionList } from '@elastic/eui';
+import type { PersistableStateAttachmentViewProps } from '@kbn/cases-plugin/public/client/attachment_framework/types';
+import type { CoreStart } from '@kbn/core/public';
+import { getAnomalySwimLaneEmbeddableComponent } from '../embeddables/anomaly_swimlane';
+import type { MlStartDependencies } from '../plugin';
+import { PLUGIN_ICON } from '../../common/constants/app';
+import { ANOMALY_SWIMLANE_EMBEDDABLE_TYPE, AnomalySwimlaneEmbeddableInput } from '../embeddables';
+
+const AttachmentActions: React.FC = () => {
+  return (
+    <EuiButtonIcon
```

Comment:
Where is this shown? The `aria-label` should be internationalized if this is shown to the user.